src/libraries/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -97,19 +96,36 @@ public async Task SendAsync_SlowServerAndCancel_ThrowsTaskCanceledException()
             }
         }
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/20675")]
         [OuterLoop]
         [Fact]
-        [OuterLoop]
-        public void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpRequestException()
+        public async void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpRequestException()
         {
             var handler = new WinHttpHandler();
             using (var client = new HttpClient(handler))
             {
-                Task<HttpResponseMessage> t = client.GetAsync(SlowServer);
+                var triggerResponseWrite = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
+                var triggerRequestWait = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
 
-                AggregateException ag = Assert.Throws<AggregateException>(() => t.Wait());
-                Assert.IsType<HttpRequestException>(ag.InnerException);
+                await LoopbackServer.CreateServerAsync(async (server, url) =>
+                {
+                    Task serverTask = server.AcceptConnectionAsync(async connection =>
+                    {
+                        await connection.SendResponseAsync($"HTTP/1.1 200 OK\r\nDate: {DateTimeOffset.UtcNow:R}\r\nContent-Length: 16000\r\n\r\n");
+
+                        triggerRequestWait.SetResult(true);
+                        await triggerResponseWrite.Task;
+                    });
+
+                    AggregateException ag = await Assert.ThrowsAsync<AggregateException>(async () =>
+                    {
+                        Task<HttpResponseMessage> t = client.GetAsync(url);
+                        await triggerRequestWait.Task;
+                        t.Wait();
```

Comment:
Why `t.Wait();` and not `await`?

Reply:
I kept the original code here, it makes sense to change it to `await` now since it's an `async` function now. Thanks!